httpref.go

```diff
@@ -40,7 +41,13 @@ func (r References) InRange(code string) References {
 	for _, v := range r {
 		if strings.Contains(v.Name, "-") {
 			parts := strings.Split(v.Name, "-")
-			if code >= parts[0] && code <= parts[len(parts)-1] {
+			if len(parts) != 2 {
+				fmt.Printf("Invalid range format for reference name: %s\n", v.Name)
+				continue
+			}
+
+			start, end := parts[0], parts[1]
+			if code >= start && code <= end {
```

Comment:
There should be an additional test for this behaviour if something is changing.

Reply:
> There should be an additional test for this behaviour if something is changing.

well I will add some sort of tests for an edge case at httpref_test.go file